lib/cluster-providers/ec2-cluster-provider.ts

```diff
@@ -74,6 +74,13 @@ export interface EC2ProviderClusterProps extends CommonClusterOptions {
      * Select either SPOT or ON-DEMAND
      */
     nodeGroupCapacityType?: CapacityType;
+
+    /**
+     * Is it a private only EKS Cluster
+     *
+     * @default true
```

Comment:
I think the default should stay private and public with an option to override.

Reply:
Let's resolve this, otherwise lgtm, nice!

Reply:
Resolved.